consensus/polybft/data_types.go

```diff
@@ -0,0 +1,160 @@
+package polybft
```

Comment:
Can we move this back to `state.go`?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/785/commits/56b13fd4757f4df9028d7c70d695afd781f389a6